### ISSUE TITLE
Deprecation Warning

### ISSUE THREAD

**Issue by seansellek** [workmanw/ember-diff-attrs#8]
We are seeing the following deprecation warning:

```
DEPRECATION: An addon is trying to access project.nodeModulesPath. This is not a reliable way to discover npm modules. Instead, consider doing: require("resolve").sync(something, { basedir: project.root }). Accessed from:   DependencyVersionChecker.NPMDependencyVersionChecker (/Users/ssellek/Impinj/new-imc/threshold-monitor/node_modules/ember-diff-attrs/node_modules/ember-cli-version-checker/src/npm-dependency-version-checker.js:11:32)
```

while running `ember -v`:

```
ember-cli: 2.18.0
node: 6.12.3
os: darwin x64
```

[This issue](https://github.com/ember-cli/ember-cli-version-checker/issues/48) indicates that versions of `ember-cli-version-checker` < 2.0.0 throw this warning.

` npm ls ember-cli-version-checker` results in: 

```
├─┬ ember-cli-babel@6.11.0
│ └── ember-cli-version-checker@2.1.0
└─┬ ember-flatpickr@2.3.0
  └─┬ ember-diff-attrs@0.2.0
    └─┬ ember-cli-babel@5.2.4
      └── ember-cli-version-checker@1.3.1
```

It appears `ember-diff-attrs` is using a very outdated version of `ember-cli-babel`, which lists a sub-2.0 version of `ember-cli-version-checker` as a dependency, triggering this deprecation warning.

**Comment by workmanw** [comment]
I think we just need to update this addon. Give me a few.

**Comment by workmanw** [comment]
So a few turned into "a few days", but I just published v0.2.1. Thanks for your help.
